test/gtest/ucp/ucp_test.cc

```diff
@@ -729,3 +729,109 @@ void ucp_test_base::entity::ep_destructor(ucp_ep_h ep, entity *e)
     EXPECT_EQ(UCS_OK, status);
     ucp_request_release(req);
 }
+
+ucp_test::mapped_buffer::mapped_buffer(size_t size,
+                                       const entity& entity,
+                                       int flags,
+                                       uint64_t seed,
+                                       ucs_memory_type_t mem_type) :
+    m_entity(entity),
+    m_buffer((flags & UCP_MEM_MAP_FIXED) ? ucs::mmap_fixed_address() : NULL,
```

Comment:
add static helper function mapped_buffer::alloc_address(flags) to do `(flags & UCP_MEM_MAP_FIXED) ? ucs::mmap_fixed_address() : NULL`

Reply:
added alloc_address